packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -401,6 +402,7 @@ registerFragment(`
       htmlHighlight
       wordCount
       version
+      plaintextDescription
```

Comment:
I don't think this is necessary here, right? This is for SunshiePostsList, which is for the moderation sidebar. (Sunshine Regiment is LW's name for moderators)